app/controller/settings.js

```diff
@@ -0,0 +1,27 @@
+var config = module.exports = {};
+
+var fs = require('fs');
+
+config.data = [];
+
+config.update = function (req) {
+	var data = config.get();
+	var newConfig = config.set(req);
+	data[newConfig.id - 1] = newConfig;
+	return data;
+};
+
+config.get = function() {
+	var data = config.data;
+	data = require('../config.json');
+	return data;
```

Comment:
this is backwards `var data = require('../config.json');` should be first then `config.data = data;`